setup.cfg

```diff
@@ -102,8 +106,8 @@ testpaths =
     docs src tests
 
 [coverage:run]
-# This is covered in the versiongit test suite so exclude it here
-omit = */_version_git.py
+# Ignore generated _version.py
+omit = src/*/_version.py
```

Comment:
If we delete this line do we still get the same coverage result? If so, it can go

Reply:
It does not stay the same. We get 2 more statements, but they are covered.

Reply:
That sounds fine to me